Project_0.Altis/Misc/fn_actionDropAllWeaponsAndSurrender.sqf

```diff
@@ -11,7 +11,9 @@ Author: Sen 01.03.2019
 */
 params ["_unit"];
 
-private _currentWeapon = currentWeapon _unit;
+_unit setCombatMode "BLUE";
```

Comment:
I am pretty sure that setCombatMode is a group setting.
https://community.bistudio.com/wiki/setCombatMode

Reply:
Often when a group is requested for a function unit can be used as well(unit cant exist without group).